### ISSUE TITLE
How to Convert WIDER_Face to .tfrecord?

### ISSUE THREAD

**Issue by wfchenwf** [xggIoU/centernet_tensorflow_wilderface_voc#2]
features = {"img_raw_b": tf.FixedLenFeature([], tf.string),  # 定长特征解析
                "img_raw_g": tf.FixedLenFeature([], tf.string),
                "img_raw_r": tf.FixedLenFeature([], tf.string),
               "label": tf.FixedLenFeature([],tf.string),
                "width": tf.FixedLenFeature([], tf.int64),
                "height": tf.FixedLenFeature([], tf.int64)}
How are the parameters of these features calculated?
Can you share the conversion file, thank you

**Comment by xggIoU** [comment]
These days, I will update the data production after I go home. If you can't wait, you can make it in the following way：imagedata  classid xmin ymin xmax ymax.
if you detect face,classid is 0.

**Comment by wfchenwf** [comment]
Ok, thanks again，i will try it.




------------------ 原始邮件 ------------------
发件人: "xggIoU"<notifications@github.com>;
发送时间: 2019年6月12日(星期三) 中午11:22
收件人: "xggIoU/centernet_tensorflow_wilderface_voc"<centernet_tensorflow_wilderface_voc@noreply.github.com>;
抄送: "陈文峰"<961403391@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [xggIoU/centernet_tensorflow_wilderface_voc] How to ConvertWIDER_Face to .tfrecord? (#2)




These days, I will update the data production after I go home. If you can't wait, you can make it in the following way：imagedata  classid xmin ymin xmax ymax.
 if you detect face,classid is 0.

**Comment by ahuyzx** [comment]
Can you share the code of how to create tfrecord? Thank you very much.

**Comment by xggIoU** [comment]
> Can you share the code of how to create tfrecord? Thank you very much.



> Ok, thanks again，i will try it.
> […](#)
> ------------------ 原始邮件 ------------------ 发件人: "xggIoU"<notifications@github.com>; 发送时间: 2019年6月12日(星期三) 中午11:22 收件人: "xggIoU/centernet_tensorflow_wilderface_voc"<centernet_tensorflow_wilderface_voc@noreply.github.com>; 抄送: "陈文峰"<961403391@qq.com>;"Author"<author@noreply.github.com>; 主题: Re: [xggIoU/centernet_tensorflow_wilderface_voc] How to ConvertWIDER_Face to .tfrecord? (#2) These days, I will update the data production after I go home. If you can't wait, you can make it in the following way：imagedata classid xmin ymin xmax ymax. if you detect face,classid is 0. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.
Sorry, I’m a little busy lately, I’ve updated the code for creating data under folder img2tfrecords_detection.

**Comment by xggIoU** [comment]
Sorry, I’m a little busy lately, I’ve updated the code for creating data under folder img2tfrecords_detection.

**Comment by ahuyzx** [comment]
I have seen it. Thank you for your sharing.
Best regards.
Zhixiang Yin

**Comment by wfchenwf** [comment]
Can you share the label of WIDER_Face, the official label does not match your format?

**Comment by xggIoU** [comment]
I have uploaded wilderface label.

**Comment by wfchenwf** [comment]
Thank you so much.

**Comment by wfchenwf** [comment]
PS D:\pythoncode\centernet_tensorflow_wilderface_voc-master> python .\train.py
WARNING:tensorflow:From D:\Python3.5\lib\site-packages\tensorflow\python\framework\op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From D:\pythoncode\centernet_tensorflow_wilderface_voc-master\shufflenetv2_centernet.py:80: conv2d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.conv2d instead.
WARNING:tensorflow:From D:\Python3.5\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
2019-06-24 16:24:15.608201: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2019-06-24 16:24:15.851322: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties:
name: GeForce GTX 1060 major: 6 minor: 1 memoryClockRate(GHz): 1.733
pciBusID: 0000:01:00.0
totalMemory: 6.00GiB freeMemory: 4.97GiB
2019-06-24 16:24:15.859099: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-06-24 16:24:16.248122: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-06-24 16:24:16.252627: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0
2019-06-24 16:24:16.255085: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N
2019-06-24 16:24:16.258433: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 4714 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1060, pci bus id: 0000:01:00.0, compute capability: 6.1)
PS D:\pythoncode\centernet_tensorflow_wilderface_voc-master>

When I run train.py, the program quits without error, could you help me?
